elastic/security/templates/composable/.logs-endpoint.action.responses.json

```diff
@@ -21,6 +21,9 @@
       }
     },
     "composed_of": [
+      "logs@mappings",
```

Comment:
I see standard component templates are added here and used by all existing templates...I don't see them as part of the track. This means that we rely on KIbana to install them?

Reply:
AFAIU the component templates added are installed by ES.

Reply:
Yes, install these particular templates and with Kibana included in the security track we do not have a mechanism to  distinguish if these templates are not installed if Kibana is not used.